apps/api/src/app/change/usecases/apply-change/apply-change.usecase.ts

```diff
@@ -19,9 +19,7 @@ export class ApplyChange {
     });
 
     const changes = await this.changeRepository.find(
-      {
-        _parentId: parentChange._id,
-      },
+      { _environmentId: command.environmentId, _parentId: parentChange._id },
```

Comment:
I think it is safer to take parentChanges env id 😄

Reply:
so the update will be 
      { _environmentId: parentChange._environmentId, _parentId: parentChange._id },
right?